scripts/jest/jest-cli.js

```diff
@@ -0,0 +1,268 @@
+'use strict';
+
+const {spawn} = require('child_process');
+const chalk = require('chalk');
+const yargs = require('yargs');
+
+const configMap = {
+  oss: './scripts/jest/config.source.js',
+  www: './scripts/jest/config.source-www.js',
+  devtools: './scripts/jest/config.build-devtools.js',
+};
+
+// TODO: These configs are separate but should be rolled into the configs above
+// so that the CLI can provide them as options for any of the configs.
+const persistentConfig = './scripts/jest/config.source-persistent.js';
+const buildConfig = './scripts/jest/config.build.js';
+
+const argv = yargs
+  .parserConfiguration({
+    // Important: This option tells yargs to move all other options not
+    // specified here into the `_` key. We use this to send all of the
+    // Jest options that we don't use through to Jest (like --watch).
+    'unknown-options-as-args': true,
+  })
+  .wrap(yargs.terminalWidth())
+  .options({
+    debug: {
+      alias: 'd',
+      describe: 'Run with node debugger attached.',
+      requiresArg: false,
+      type: 'boolean',
+      default: false,
+    },
+    project: {
+      alias: 'p',
+      describe: 'Run the given project.',
+      requiresArg: false,
```

Comment:
Shouldn't this be true?

Reply:
We default to the OSS project so that `yarn test` runs the OSS config like it does today. Same for release channel, where we default to `experimental` like the configs do.

Reply:
Right, a default value makes sense- but passing the `--project` arg without any value (which is what `requiresArg` suggests) doens't.

Reply:
Oh I misunderstood what `requiresArg` was, thanks for catching it!